node/index.ts

```diff
@@ -1,12 +1,21 @@
-import {hrToMillis, HttpClient, Logger, MetricsAccumulator} from '@vtex/api'
+import {Apps, hrToMillis, HttpClient, Logger, MetricsAccumulator} from '@vtex/api'
 
+import * as Bluebird from 'bluebird'
 import {map} from 'ramda'
 
 import {robots} from './middlewares/robots'
 import {sitemap} from './middlewares/sitemap'
+import {userSitemap} from './middlewares/userSitemap'
+import {customSitemap} from './middlewares/customSitemap'
 
 (global as any).metrics = new MetricsAccumulator()
 
+global.Promise = Bluebird
```

Comment:
try using bluebird-global modules instead of setting the global promise manually

Reply:
Couldn't make this work. I'll leave this as it is for now.

Reply:
It's odd, acording to [here](https://github.com/kaatt/bluebird-global/blob/master/index.js), it's doing the same attribution. But ok, let's not spend time on this kind of thing